### PR TITLE
FUSETOOLS-2421 - better clean up for JMX Remote Route Edit IT

### PR DESCRIPTION
**Pull request by lhein**

### CODE REVIEW

File: jmx/tests/org.fusesource.ide.jmx.camel.tests.integration/src/org/fusesource/ide/jmx/camel/tests/integration/navigator/RemoteDebugWhenEditingRoutesFromJMXNavigatorIT.java [hunk 1]
```diff
@@ -57,11 +57,14 @@ public void tearDown() throws Exception {
...
 		}
 		projectWithDebugAvailableDeployedHelper.clean();
+		if (remoteDebuglaunch != null) {
+			remoteDebuglaunch.terminate();
+		}
 	}
 	
 	@Test
 	public void testRemoteDebugScenario() throws Exception {
-		DefaultConnectionWrapper jmxConnection = initializeConnection();
```
Comment (apupier) [inline jmx/tests/org.fusesource.ide.jmx.camel.tests.integration/src/org/fusesource/ide/jmx/camel/tests/integration/navigator/RemoteDebugWhenEditingRoutesFromJMXNavigatorIT.java]: if setting the attribute here then remove the set of the attribute on line 81

File: jmx/tests/org.fusesource.ide.jmx.camel.tests.integration/src/org/fusesource/ide/jmx/camel/tests/integration/navigator/RemoteDebugWhenEditingRoutesFromJMXNavigatorIT.java [hunk 2]
```diff
@@ -57,11 +57,14 @@ public void tearDown() throws Exception {
 			ExtensionManager.getProvider(DefaultConnectionProvider.PROVIDER_ID).removeConnection(jmxConnection);
 		}
 		projectWithDebugAvailableDeployedHelper.clean();
+		if (remoteDebuglaunch != null) {
+			remoteDebuglaunch.terminate();
```
Comment (apupier) [inline jmx/tests/org.fusesource.ide.jmx.camel.tests.integration/src/org/fusesource/ide/jmx/camel/tests/integration/navigator/RemoteDebugWhenEditingRoutesFromJMXNavigatorIT.java]: what do you think to also try to kill processes retrieved by remoteDebuglaunch.getProcesses() ?

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
PR passed on old CCI @apupier

**Comment by lhein** [comment]
closing as outdated

**Pull request closed without merging**